src/md/release.md

```diff
@@ -201,6 +242,23 @@ git push apache
 ```
 Merge `@since` tag and change log changes to `master`.
 
+### Documentation
+
+The documentation promotion is done in two steps. First the release documentation component is made the latest and then the remaining website documentation is deployed.
+
+1. In the apex-site repository run the following commands to make the release documentation the latest
+
+```
+cd docs
+# Set the release version to be the latest available version
+ln -nsf ${DOC_NAME}-${rv} ${DOC_NAME}
+git add -A
+git commit -m "Promoting ${DOC_NAME}-${rv} documentation"
+git push
+```
```

Comment:
We need to remove the previous release candidate docs folders before officially publishing apex-MAJOR.MINOR docs.